vkconfig/layer_info.json

```diff
@@ -135,14 +141,19 @@
                 "type": "bool",
                 "default": true
             },
+            "show_timestamp": {
+                "name": "Show Timestamp",
+                "description": "Show the timestamp of function calls since start in microseconds",
+                "type": "bool",
+                "default": true
```

Comment:
defaulting this to `true` will change existing behavior, right?  Is this intended?

Reply:
as discussed in #800, the ability to see the timing was thought to be useful enough to have on by default, with the option to turn it off when doing diffs. This could be added to the "hide address" option, which is the main reason to turn it off

Reply:
The defaults have been changed to false, to preserve existing behavior.
While api_dump isn't meant as a profiler, it is at least nice to have the option to see timing.

Reply:
My opinion is that it should be off by default, so that scripts that process api dump output are not broken.